scripts/remark-directives.fixtures.js

```diff
@@ -168,7 +168,9 @@ ${
 }</code-group>
 <h2>bbb</h2>`
 }
-
+/*
+  How does this demonstrate the difference between JS and TS? It looks like CommonJS vs ES6 import syntax
+*/
```

Comment:
https://www.loom.com/share/46ad05564a31425bb6dcbaf9f991987b

Reply:
The cjs vs esm import syntax is the primary difference in this sample.